sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -480,14 +484,12 @@ private[hive] class TestHiveSparkSession(
     hiveQTestUtilTables.foreach(registerTestTable)
   }
 
-  private val loadedTables = new collection.mutable.HashSet[String]
-
-  def getLoadedTables: collection.mutable.HashSet[String] = loadedTables
+  def getLoadedTables: collection.mutable.HashSet[String] = sharedState.loadedTables
 
   def loadTestTable(name: String) {
-    if (!(loadedTables contains name)) {
+    if (!(sharedState.loadedTables contains name)) {
```

Comment:
PS I wouldn't mind avoiding postfix notation here. I'm trying to remove it from the code base.

Reply:
Fixed